org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -148,6 +150,12 @@ public SpaceMode DistanceSpaceMode
         [Tooltip("Speed for extending the moving button visuals when selected by a non-touch source.")]
         private float extendSpeed = 0.5f;
 
+        [field: SerializeField, Tooltip("Enables (true) or disables (false) the Canvas Rounded Rect on hover only.")]
+        public bool EnableOnHoverOnlyCanvasRoundedRect { get; set; } = false;
+
+        [field: SerializeField, Tooltip("Enables (true) or disables (false) the Front plate Raw Image on hover only.")]
+        public bool EnableOnHoverOnlyFrontPlateRawImage { get; set; } = false;
+
```

Comment:
Wouldn't adding such visual concepts directly to PressableButton break the philosophy of this component, in that it (from \<remarks\> above) /// This script does not make any assumptions about the visuals associated with this button.

Reply:
I tend to agree with this.  Can enabling and disabling be controlled event a event handler within the prefab itself?

Reply:
> I tend to agree with this. Can enabling and disabling be controlled event a event handler within the prefab itself?

I don't understand what you mean by this.  Can you provide an example of what you mean?

Reply:
Thank you all for your feedback.  I have switched from OnHover* events to proximity interactor similar to how proximity hover lights work.  Would you agree with this new approach?